### PR TITLE
Enable hcx

### PR DESCRIPTION
**Pull request by khavinshankar**

### CODE REVIEW

**Comment by khavinshankar** [comment]
Can be closed in favor of https://github.com/coronasafe/care_fe/pull/8178, HCX will be enabled by default in develop when the communications flow is merged.

**Comment by khavinshankar** [comment]
Closing this in favor of #8178

**Comment by khavinshankar** [comment]
Temporarily reopening this PR until HCX Plug is merged

File: netlify.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ NODE_OPTIONS = "--max_old_space_size=4096"
 
 [[redirects]]
 from = "/api/*"
-to = "https://careapi.ohc.network/api/:splat"
+to = "https://work.khavinshankar.dev/api/:splat"
```
Comment (khavinshankar) [inline netlify.toml]:
```suggestion
to = "https://8k-work.khavinshankar.dev/api/:splat"
```

**Comment by bodhish** [comment]
Not sure if we need this anymore. Closing this PR due to lack of recent progress

**Pull request closed without merging**